### ISSUE TITLE
pager-edge problem

### ISSUE THREAD

**Issue by nxjosephofficial** [jekyll/minima#681]
Hello. I don't understand why it is looking like this.
![2022-10-16_12-43](https://user-images.githubusercontent.com/61632416/196028809-8e91617f-703a-463a-8221-2122c2a8de5e.png)
I'm using `minima 2.5.1`, `jekyll-paginate 1.1.0` and `jekyll 4.2.2`

**Comment by ashmaroli** [comment]
@nxjosephofficial The issue you're referring to is more related to Minima than the pagination plugin. Therefore, I have transferred the issue to Minima repository.

For further clarification, you're not using Minima 2.5, but a "repository version" of the theme using the `jekyll-remote-theme` plugin.

To help understand what exactly is your issue, I would need more details from you:
- Is it the misalignment of numbers?
- Is it the dotted square box to the left end?
- Is it the sharp edge of the left box but rounded edge of the right box?

**Comment by nxjosephofficial** [comment]
> The issue you're referring to is more related to Minima than the pagination plugin. Therefore, I have transferred the issue to Minima repository.

Okay, thank you.

> you're not using Minima 2.5, but a "repository version" of the theme using the jekyll-remote-theme plugin.

These are my plugins.
```
plugins:
 - jekyll-feed
 - jekyll-seo-tag
 - jekyll-sitemap
 - jekyll-paginate
```

> Is it the dotted square box to the left end?

Yes, definitely.

**Comment by ashmaroli** [comment]
We haven't made an official release of the theme containing skins, so you're still using the "repository version".
Regardless, to solve your issue have the following at end of your stylesheet:
```css
.pager-edge { display: none }
```

**Comment by nxjosephofficial** [comment]
Do using official release instead of repository version matter to appearing of dotted square box?

I added it like this.
```
      &.pager-edge {
        color: $border-color-01;
        border: 1px dashed;
        display: none;
```

Now it looks like this.
![2022-10-16_12-43](https://user-images.githubusercontent.com/61632416/196032362-b538f91b-ae98-443b-81d2-dcc1a506d12c.png)

**Comment by ashmaroli** [comment]
> Do using official release instead of repository version matter to appearing of dotted square box?

Technically, the repository version is unstable and subject to change. The released v2.5.1 doesn't have extra styling for pager buttons.

Either ways, the issue is all CSS. You're free to make changes to suit your taste.
